addons/mail/static/src/components/attachment_viewer/attachment_viewer.scss

```diff
@@ -42,22 +47,27 @@
     display: flex;
     height: $o-navbar-height;
     align-items: center;
-    padding: 0 15px;
     width: 100%;
 }
 
 .o_AttachmentViewer_headerItem {
-    margin: 0 5px;
+    display: flex;
+    align-items: center;
+    align-self: stretch;
 
-    &:first-child {
-        margin-left: 0;
+    &:not(.o_AttachmentViewer_headerItemButton) {
+        padding: 0 map-get($spacers, 2);
     }
 
-    &:last-child {
-        margin-right: 0;
+    &:first-child {
+        margin-inline-start: map-get($spacers, 4);
```

Comment:
I would reduce this to 3, 4 still feels too much compared to other spacers used in the header.

Reply:
Actually I think the problem is that this is combined with the (left)padding on `o_AttachmentViewer_headerItem` that makes the whole space on the left of the header really big.

In the old code the icon does not have a left padding, so the spacer 4 just by itself is more acceptable.

Reply:
Done (ensured no stacking padding and margin)